File: Domain/Models/BeerData.cs
namespace Domain.Models
{
    using System;
    using Newtonsoft.Json;

    public class BeerData
    {
        [JsonProperty("id")]
        public string Id { get; set; }

        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("nameDisplay")]
        public string NameDisplay { get; set; }

        [JsonProperty("abv")]
        public string Abv { get; set; }

        [JsonProperty("style")]
        public BeerStyle Style { get; set; }

        [JsonProperty("labels")]
        public BeerLabels Labels { get; set; }

        [JsonProperty("createDate")]
        public DateTime CreateDate { get; set; }

        [JsonProperty("updateDate")]
        public DateTime UpdateDate { get; set; }

        [JsonProperty("isOrganic")]
        public char IsOrganic { get; set; }

        [JsonProperty("isRetired")]
        public char IsRetired { get; set; }
    }
}

File: TaganiWineCellarProject/ClientApp/src/app/models/beer-data.ts
import { BeerLabel } from './beer-label';
import { BeerStyle } from './beer-style';

export class BeerData {
    id: string;
    name: string;
    nameDisplay: string;
    abv: number;
    createDate: Date;
    updateDate: Date;
    isOrganic: string;
    isRetired: string;
    labels: BeerLabel;
    style: BeerStyle;
}
